### ISSUE TITLE
line 133: WARNING: unbound variable

### ISSUE THREAD

**Issue by alistair-broomhead** [xdissent/ievms#245]
When I attempt to run the script as shown in the README.md I get the following:

```
hp-mint al # curl -s https://raw.githubusercontent.com/xdissent/ievms/master/ievms.sh | bash
Checking for VirtualBox
Checking for Oracle VM VirtualBox Extension Pack
bash: line 133: WARNING: unbound variable
hp-mint al # 
```

I am running the latest stable version of Mint under Xfce

**Comment by adborden** [comment]
Same error. I'm using GNU bash, version 4.3.30(1)-release (x86_64-pc-linux-gnu)

**Comment by adborden** [comment]
Looks like the error message changed and it's not getting matched.

```
ERROR: WARNING: The VirtualBox kernel modules are not loaded.
         Please load all the needed kernel modules by:

           for m in vbox{drv,netadp,netflt}; do modprobe $m; done

         You will not be able to start VMs until this problem is fixed.
4.3.18_Gentoor96516
```

I've added a check for the new string, but I wonder if we should be more loose in our checking. e.g. Anything with a newline is probably an error.

**Comment by xdissent** [comment]
Good catch, sorry I didn't see this sooner. We just switched to VirtualBox 5.0, so I'll double check the err message there and get this fixed.

**Comment by Potherca** [comment]
Is there any news on this?

**Comment by gangsthub** [comment]
same output here :anguished:

**Comment by ikodev** [comment]
Same for me (ubuntu 14.04), I found a solution.
The script tried to get the available versions with 
`VBoxManage -v`
However this command fails and the script crash on line 133 (outbound variable).
The first command asks me to install virtualbox-dkms.
`sudo apt-get install virtualbox-dkms`

No more problem :)

**Comment by csi-lk** [comment]
Hmm tried @ikodev 's solution but didn't work for me - Ubuntu 17.04 / VirtualBox 5.1.22_Ubuntu r115126

```
virtualbox-dkms is already the newest version (5.1.22-dfsg-0ubuntu1.17.04.1).
virtualbox-dkms set to manually installed.
```

Cheers
